webui/settings.py

```diff
@@ -25,7 +25,10 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
```

Comment:
I think with code and comments above we can remove this warning

Reply:
Agree